### PR TITLE
Immediate dispatch

### PR DESCRIPTION
**Pull request by andreasohlund**
@Particular/documentation please review

### CODE REVIEW

**Comment by andreasohlund** [comment]
@SzymonPobiega can you give this one a pre-review and see if you like the terminology?

File: nservicebus/messaging/immediate-dispatch.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+NOTE: By immediate dispatch outgoing message are not [batched](/nservicebus/messaging/batched-dispatch.md) or enlisted in the current receive transaction even if the transport has support for it.
+
+### Suppressing the transaction scope
+
+Version 6 and below allows you to suppress the ambient transaction in order to have the outgoing operation being sent immediately.
+
+<!-- import RequestImmediateDispatchUsingScope -->
+
+The issue with this approach is that it only works for transports that enlists the receive operation in a transaction scope. Currently this would be MSMQ and SqlServer in DTC mode. Should you use any other transport or disable the DTC this no longer work and the outgoing message might be rolled back together with the incoming message.
```
Comment (SeanFeldman) [inline nservicebus/messaging/immediate-dispatch.md]: "this no longer work" - should be "works"

File: nservicebus/messaging/send-a-message.md [hunk 2]
```diff
@@ -9,16 +9,31 @@ redirects:
...
  Or instantiate and send all messages at once:
 
-```C#
-bus.Send<IMyMessage>( m => { m.Prop1 = v1; m.Prop2 = v2; });
-```
+<!-- import BasicSendInterface -->
 
+## Batched Dispatch
+
+While its usually best to let NServiceBus [handle exceptions for you](/nservicebus/errors) there are some scenarios where you want to send messages out even though the incoming message is rolled back. One example would be sending a reply notifying that there was an issue processing the message.
```
Comment (SzymonPobiega) [inline nservicebus/messaging/send-a-message.md]: ...even though *receiving* of the...

**Comment by SzymonPobiega** [comment]
@andreasohlund I did minor fixes.

**Comment by andreasohlund** [comment]
This one is now good to go

**Comment by DavidBoike** [comment]
:+1:

**Pull request merged**